chrome/content/OutOfOfficeMenu.js

```diff
@@ -121,5 +128,4 @@ window.addEventListener("load", function () { window.removeEventListener("load",
 	{
 		openContentTab("about:memory", "tab", "www.1st-setup.nl");
 	},
-}*/
-
+}*/
```

Comment:
Looks like all trailing line breaks were removed. Personally I find it more consistent when every line of code (including the last one) ends with a line break (`\n`). Can you configure the formatter to keep (and - if needed - add) those?

Reply:
Indeed, the javascript formatter has such an option (although, not the XML one), I'm adding it to the Makefile in another pull request. I'm closing this one to do another clean pull request.